src/components/menu/SubmenuItem.scss

```diff
@@ -10,15 +10,16 @@
         top: $bdl-submenu-offset;
         right: auto;
         left: calc(100% - #{$bdl-submenu-offset});
+        z-index: 9999;
 
         &.is-left-aligned {
             right: 100%;
             left: auto;
         }
 
         &.is-hidden {
-            right: auto;
-            left: -9999px;
+            right: auto !important;
+            left: -9999px !important;
```

Comment:
We try to avoid the use of `!important` at all times. Can we determine what's causing the specificity issue and resolve it as the source, instead?